.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: cpp
+sudo: false
+
+env:
+  global:
+    - NODE_VERSION="6.1.0"
+
+os:
+  - linux
+  - osx
+
+before_install:
+  - rm -rf ~/.nvm
+  - git clone https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm --version
+  - nvm install $NODE_VERSION
+  - node --version
+  - npm --version
+  - pip install --user cpplint
+
+install:
+  - npm install --build-from-source
```

Comment:
Ah, cool. Let me check that. I had to fallback to nvm because it wasn't working for OS X at some point, but let's see.